### PR TITLE
Add stakewise space, skin and custom domain

### PR DESCRIPTION
**Pull request by tsudmi**
#### Hi! What is your PR about?

- [x] Add images for your space
- [x] Add custom domain for your space
- [ ] Add alias for migrate your space
- [x] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: skins/stakewise.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  }
+
+  #topnav a {
+    color: white !important;
+  }
+
+  #topnav button {
+    border-color: white;
+    color: white;
+  }
```
Comment (gokaiorg) [inline skins/stakewise.scss:24]: Please use only theme variables. No extra custom styles are allowed except on #topnav 
```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/stakewise.eth

File: skins/stakewise.scss (hunk 1 above)
Comment (tsudmi) [reply skins/stakewise.scss:24]: Hey, the styles I've provided here were adopted from the [yearn skin](https://github.com/snapshot-labs/snapshot-spaces/blob/master/skins/yearn.scss) that has those variables specified as well. Is it not allowed to specify them anymore?
They change the style of `#topnav`, so it seems to comply with the rule.

**Comment by tsudmi** [comment]
@gokaiorg I've fixed the space, now it should be accessible at https://snapshot.org/#/stakewise.eth.

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**